app/src/main/java/com/ecosystem/kin/app/AccountInfoServiceImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.ecosystem.kin.app;
+
+import com.ecosystem.kin.app.model.SignInRepo;
+import com.kin.ecosystem.Kin;
+import com.kin.ecosystem.common.KinCallback;
+import com.kin.ecosystem.common.exception.KinEcosystemException;
+import com.kin.ecosystem.common.model.WhitelistData;
+import com.kin.ecosystem.transfer.AccountInfoService;
+/*
+*  Example usage of extending AccountInfoService
+*  Name of class should not be changed
+*
+* */
+public class AccountInfoServiceImpl extends AccountInfoService {
+    public AccountInfoServiceImpl() {
+        super("AccountInfoService");
+    }
```

Comment:
So only the service name should not be changed.
Can we have it in the SDK module? so the developer doesn't need to do anything.